src/main/java/ch/saunah/saunahbackend/service/GoogleCalendarService.java

```diff
@@ -124,6 +127,33 @@ public String insertEvent (String calenderID , Event event) throws IOException {
         event = service.events().insert(calenderID, event).execute();
         return event.getId();
     }
+
+    /**
+     * Add a google calender Event that is tentative with a Booking
+     * @param calenderID Id of the google Calender
+     * @param booking A booking from the sauna
+     * @return The Google calender event id
+     * @throws IOException
+     */
+    public String createEvent(String calenderID ,Booking booking) throws IOException{
+        Event event = new Event()
+            .setSummary(booking.getSaunaName())
+            .setLocation(booking.getLocation())
+            .setDescription(booking.getSaunaDescription());
```

Comment:
I think we should not set the location of the booking to the calendar, because calendar events can be queried publicly, which might be a bit problematic.

For the name, I would suggest to set it according to the status of the booking: e.g. if it's still unconfirmed, call it something like "Provisorisch" and if it's confirmed something like "Gebucht". That way, it's easy for customers to realise what the actual availability of the sauna is, when they see the calendar on the website.
So if a booking is updated, we would have to update the event name again.

And I think we can leave away the description, as this is not really needed in the event.